api/v1beta2/ibmvpccluster_types.go

```diff
@@ -108,6 +108,20 @@ type VPCLoadBalancerSpec struct {
 	Subnets []VPCResource `json:"subnets,omitempty"`
 }
 
+// AdditionalListenerEnabled defines what category of VMs are added to the listener pool.
+type AdditionalListenerEnabled int
+
+const (
+	// AdditionalListenerAll add every VM to the listener pool.
```

Comment:
Using 0, 1 & 2 to identify the nodes does not look good to me. Can we define the possible values using enum like [this](https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/blob/main/api/v1beta2/types.go#L138) and add the necessary kubebuilder validation?